rules/flow.js

```diff
@@ -15,23 +15,40 @@ module.exports = {
     'flowtype/delimiter-dangle': ['error', 'always-multiline', 'always-multiline', 'always-multiline'],
     'flowtype/generic-spacing': ['error', 'never'],
     'flowtype/newline-after-flow-annotation': ['error', 'always'],
-    'flowtype/object-type-delimiter': ['error', 'comma'],
     'flowtype/no-mixed': 'off',
     'flowtype/no-dupe-keys': 'error',
+    // TODO: enable?
+    'flowtype/no-existential-type': 'off',
+    'flowtype/no-flow-fix-me-comments': 'off',
+    'flowtype/no-mixed': 'off',
+    // TODO: enable?
+    'flowtype/no-mutable-array': 'off',
```

Comment:
We have this 

> Note that initialization of a variable with an empty array is considered valid (e.g., const values: Array<string> = [];). This behavior resembles the behavior of Flow's unsealed objects, as it is assumed that empty array is intended to be mutated.

And also we will still be able to do perf optimisations using `// eslint-disable-next-line` it will emphasis the alert on the mutation